packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -463,7 +483,12 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
 
     private navigateToStep(type: CheckoutStepType, options?: { isDefault?: boolean }): void {
```

Comment:
If a shopper has a saved address and skips directly to the credit card info step, does `navigateToStep` fire for the "customer", "shipping address" and "billing address" steps?

Reply:
I've tested for this and yes, it fires for each step that there is saved info for